docs/getting-started/gantt-chart-introduction/README.md

```diff
@@ -27,35 +27,33 @@ To find out more about the functionalities for Gantt charts, please visit our [u
 
 ## What is a Gantt chart?
 
-With Gantt charts in OpenProject you can create and manage a project plan and share the information with your team. You can schedule your tasks and visualize the required steps to complete your project. As a project manager you are directly informed about delays in your project and can act accordingly.
+With Gantt charts in OpenProject you can create and manage project plans and share the information with your team. You can schedule your tasks and visualize the required steps to complete your project. As a project manager you are directly informed about delays in your project and can act accordingly.
 
-The dynamic Gantt chart in OpenProject displays the phases, tasks and milestones in your project as well as relationships between them. The elements for phases and tasks each have a start and end date, so you are always informed about the current status. The milestone only appears as a fixed point in time.
+The dynamic Gantt chart in OpenProject displays the phases, tasks and milestones in your project as well as relationships between them. The elements for phases and tasks each have a start and end date, so you are always informed about the current status. The milestones only appears as a fixed point in time.
 
 ## Activate the Gantt chart view
 
-To open the Gantt chart view in OpenProject, the [work packages module needs to be activated](../../user-guide/activity) in the project settings.
+To open the Gantt chart view in OpenProject, the [Gantt charts and work packages modules need to be activated](../../user-guide/projects/project-settings/modules/) in the project settings. Both are usually activated by default.
 
-Within your project menu, navigate to the work packages module. Select the **Gantt chart view** in the work package table with the button on the top right.
+Within your project menu, navigate to the **Gantt charts** module.
 
-![open the gantt chart view](gantt-chart-view.png)
+![Select the Gantt charts module in OpenProject](openproject-user-guide-select-gantt-charts-module.png)
 
-You can activate a Gantt chart view from any work package table (or saved query) by selecting the Gantt view from the top.
+The Gantt chart then displays all work package types, e.g. phases and milestones, tasks or bugs, in a timeline view. It shows dependencies between different work packages as well as additional information, e.g. subject, start or due dates. 
 
-The Gantt chart then displays all work package types, e.g. phases and milestones, tasks or bugs, in a timeline view.
-
-It shows dependencies between different work packages as well as additional information, e.g. subject, start or due dates.
+You can also select the project overarching Gantt charts view through the [global modules](../../user-guide/home/global-modules/) menu, which will include the information, which project the work packages are part of. 
```

Comment:
Suggestion, but not critical: Perhaps "transversal" or "cross-project" instead of overarching?

Reply:
I will use cross-project in this case